ci/gpu/build.sh

```diff
@@ -79,21 +79,10 @@ gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge/label/cupy_rc -c
 gpuci_logger "Install cuCIM and its dependencies"
 
 gpuci_logger "Install dependencies"
-gpuci_conda_retry create -n cucim -y -c conda-forge -c conda-forge/label/cupy_rc -c rapidsai-nightly \
+gpuci_conda_retry install -y -c conda-forge -c rapidsai-nightly \
     rapids-doc-env \
-    flake8 \
-    pytest \
-    pytest-cov \
-    python=${PYTHON_VER} \
-    conda-forge/label/cupy_rc::cupy=9 \
-    cudatoolkit=${CUDA_VER} \
-    numpy \
-    scipy \
-    scikit-image=0.18.1 \
     openslide
```

Comment:
Submitted PR ( https://github.com/rapidsai/integration/pull/263 ) to add this to the integration package. Can remove here if we decide it is ok to add over there

Reply:
This PR has been merged and this code dropped. Marking resolved